### PR TITLE
03 - Add Order Form

### PR DESCRIPTION
**Pull request by Baavi**
It connects to the DB, and works, but needs to finalize the DB layout before  writing the query and is dependent on the error msg components to display the validation errors

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,2 +1,82 @@
...
+                <div class="inputField">
+                    <label for="product_ID1">Product ID</label>
+                    <select name="product_ID1" id="product_ID1">
+                        <option value="placeholder" hidden>Please Select</option>
+                        <option value="Milk">Milk</option>
+                        <option value="KitKat">KitKat</option>
+                        <option value="Milo">Milo</option>
+                        <option value="Cheese">Cheese</option>
+                        <option value="Ice Cream">Ice Cream</option>
+                    </select>
```
Comment (RoryHowlettUni) [inline index.php:31]: I may've missed a conversation, but were we going to have the ability to search in the drop down field too??

File: index.php [hunk 2]
```diff
@@ -1,2 +1,82 @@
...
+                <div class="inputField">
+                    <label for="quantity1">Quantity </label>
+                    <input type="number" name="quantity1" id="quantity1" required="required" onblur="show_total()" />
+                </div>
+                <div class="inputField">
+                    <label for="price1">Price </label>
+                    <input type="number" name="price1" id="price1" required="required"
+                        onblur="show_total(); show_row2();" />
+                    <br>
+                </div>
```
Comment (RoryHowlettUni) [inline index.php:42]: Any particular reason why the label and the type of field here are different to whats shown in Figma? I thought the intention was for the price to be hardcoded, although I do think being able to change the price during checkout is a good idea

Comment (RoryHowlettUni) [reply index.php:42]: The "Total Price" for each thing, which is an AC for this ticket isn't here either, was that intended?

File: index.php [hunk 3]
```diff
@@ -1,2 +1,82 @@
...
+                    <label for="price2">Price </label>
+                    <input type="number" name="price2" id="price2" onblur="show_total()" />
+                    <br>
+                </div>
+            </div>
+            <div id="row_3">
+                <div class="inputField">
+                    <br>
+                    <label for="member_id">Member ID / Email </label>
+                    <input type="text" name="member_id" placeholder=" " id="member_id" required="required" />
```
Comment (RoryHowlettUni) [inline index.php:70]: This field doesn't quite match Figma either, may want to double check that.

File: index.php [hunk 4]
```diff
@@ -1,2 +1,82 @@
...
+                    <br>
+                </div>
+            </div>
+            <div class="inputField">
+                <label id="total_price"></label></P><br>
```
Comment (RoryHowlettUni) [inline index.php:75]: Another Figma mismatchhhh

File: scripts/addorder.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+function show_row2() {
+  var r1_price = document.getElementById("price1").value.trim();
+  var r2 = document.getElementById("row_2");
+  if (r1_price != "") {
```
Comment (RoryHowlettUni) [inline scripts/addorder.js:4]: The user could (for whatever reason) just put in the price and the next row will show up despite the other fields being blank. Not sure if this is a big deal though

File: styles/addorder.css [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  padding-top: 4rem;
+  margin: auto;
+  width: fit-content;
+}
+
+#row_1,
+#row_3 {
+  display: flex;
+  flex-direction: row;
+}
```
Comment (RoryHowlettUni) [inline styles/addorder.css:90]: The styling doesn't quite match what is shown in Figma, may want to have a squizz.

File: index.php (hunk 1 above)
Comment (Baavi) [reply index.php:31]: We could, but it's a bit complicated, better keep it for later once the form is fully functioning (nice to have)

**Pull request closed without merging**